src/App.js

```diff
@@ -1,15 +1,30 @@
+import React, { useState, useEffect } from 'react'
 import SearchBar from './components/SearchBar/SearchBar';
 import Users from './components/Users/Users';
+import axios from 'axios';
 import './App.css';
 
 function App() {
+
+  const [ users, setUsers ] = useState([]);
+  const [expanded, setExpanded ] = useState([]);
+  const [ searchInput, setSearchInput] = useState('');
+  
+
+  const { id } = users;
   // TODO: Fetch data here
+  useEffect(() => {
+    axios.get('https://users-app-backend.onrender.com/users')
+    .then(res => setUsers(res.data.data))
+    .catch(err => console.log(err))
+  })
 
   return (
+
     <div className="App">
       <h1>Our Users</h1>
       <SearchBar />
-      <Users />
+      <Users users={users} expanded={expanded.includes(id)} setExpanded={setExpanded}/>
```

Comment:
I think you're close here. It looks like a good idea to pass `expanded` and `setExpanded` down to `<Users />`. However, I think it's too soon to turn the `expanded` array into a boolean. This should happen when you are rendering a single `<User />` and know that user's `id`. A better place for that is in `<Users />` when you map through the users. Here, I am guessing this line of code will throw an error because `id` is not defined?